app/assets/stylesheets/_entries.scss

```diff
@@ -128,6 +127,21 @@
 }
 
 .entries-table {
+  line-height: 20px;
+  table-layout: auto;
+
+  td,
+  th {
+    padding: 4px;
+
+    &:first-child {
+      padding-left: 0;
+    }
```

Comment:
The big problem with 2 tables is that i have to differentiate between entries everywhere because there is only 1 controller for both types of entries. for instance for creating a new entry, the form sends an extra param so that the controller knows which params to request and what table to write to. The controller then has to pick the right variable and params. This leads to a lot of if entry_type, statements. As far as i know the only way to avoid this without using 1 table would be to create a seperate controller. 
The only place where 1 list of both entries might come in usefull is when downloading the CSV.